core/templates/dev/head/components/AnswerGroupEditorDirective.js

```diff
@@ -208,9 +208,13 @@ oppia.directive('answerGroupEditor', [function() {
         };
 
         $scope.cancelActiveRuleEdit = function() {
-          $scope.rules.splice(0, $scope.rules.length);
-          for (var i = 0; i < $scope.rulesMemento.length; i++) {
-            $scope.rules.push($scope.rulesMemento[i]);
+          if ($scope.rulesMemento !== null) {
```

Comment:
Um. Given how small the change is, I reckon the fastest way is to just close this PR and make a new one on a different branch.